common/game_rules/000_wc_game_rules.txt

```diff
@@ -109,47 +109,67 @@ horde_invasion = {
 		desc = "RULE_HORDE_INVASION_STRENGTH_X0_5_DESC"
 	}
 }
-qiraji_invasion = {
-	name = "RULE_QIRAJI_INVASION_STRENGTH"
+# qiraji_invasion = {
```

Comment:
The game rule is commented but still used in the files.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/87685127-19ca5880-c794-11ea-901e-dddfd3688732.png)

</details>